server/server.js

```diff
@@ -0,0 +1,71 @@
+const express = require('express')
+const app = express()
+const port = 3333;
+
+const bodyParser = require('body-parser')
+
+const Item = require('../db/index.js')
+
+app.use(express.static('dist'))
+app.use(bodyParser.urlencoded({ extended: false }))
+app.use(bodyParser.json())
+
+app.all('/test', (req, res) => {
+  let rating = Math.floor((Math.random() * 5) + 1)
+  let hipsum = `this text is from the server`
+  let helpful = Math.floor((Math.random() * 32) + 1)
+  var singleItem = new Item({
+      reviewID: 7777,
+      author: 'testAuth',
+      rating: rating,
+      title: 'ABRA KADABRA ALAKAZAM',
+      text: hipsum,
+      helpfulCount: helpful
+  });
```

Comment:
If possible, create helper function for this in separate file

Reply:
will do later